shipyard_blueprints/ftp/shipyard_ftp/cli/delete.py

```diff
@@ -29,115 +32,63 @@ def get_args():
+        password = args.password
+        file_name_match_type = args.file_name_match_type
+        file_name = args.source_file_name
+        folder_name = shipyard.clean_folder_name(args.source_folder_name)
+
+        client = FtpClient(host=host, port=port, user=username, pwd=password)
+
+        if file_name_match_type == "regex_match":
+            folders = [folder_name]
+            files = []
+            while folders:
+                folder_filter = folders[0]
+
+                files, folders = client.find_files_in_directory(
+                    folder_filter=folder_filter, files=files, folders=folders
+                )
+
+            matching_file_names = shipyard.find_all_file_matches(
+                files, re.compile(file_name)
             )
 
-        matching_file_names = shipyard.files.find_all_file_matches(
-            files, re.compile(file_name)
-        )
-
-        number_of_matches = len(matching_file_names)
-
-        if number_of_matches == 0:
-            print(f'No matches were found for regex "{file_name}".')
-            sys.exit(ec.EXIT_CODE_NO_MATCHES_FOUND)
-
-        for index, file_name in enumerate(matching_file_names):
-            print(
-                f"Deleting file {index+1} of {len(matching_file_names)} out of {number_of_matches}"
-            )
+            number_of_matches = len(matching_file_names)
+
+            if number_of_matches == 0:
+                logger.info(f'No matches were found for regex "{file_name}".')
+                sys.exit(EXIT_CODE_NO_MATCHES_FOUND)
+
+            for index, file_name in enumerate(matching_file_names):
+                logger.info(
+                    f"Deleting file {index + 1} of {len(matching_file_names)} out of {number_of_matches}"
+                )
+                try:
+                    client.remove(file_name)
+                except Exception:
+                    logger.error(f"Failed to delete {file_name}... Skipping")
+        elif file_name_match_type == "exact_match":
+            file_path = shipyard.combine_folder_and_file_name(folder_name, file_name)
             try:
-                delete_ftp_file(client=client, file_path=file_name)
+                client.remove(file_path)
             except Exception as e:
-                print(f"Failed to delete {file_name}... Skipping")
-    else:
-        file_path = shipyard.files.combine_folder_and_file_name(folder_name, file_name)
-        try:
-            delete_ftp_file(client=client, file_path=file_path)
-        except Exception as e:
-            print(f"The server says: {e}")
-            print(
-                f"Most likely, the file name/folder name you specified has typos or the full folder name was not provided. Check these and try again."
-            )
-            sys.exit(ec.EXIT_CODE_NO_MATCHES_FOUND)
+                logger.error(
+                    "Most likely, the file name/folder name you specified has typos or the full folder name was not "
```

Comment:
I would rephrase this error message as:

"Check the file/folder name for misspellings and ensure that entire folder name was provided"

Maybe set the level to `warning` as well?

Reply:
Made the rewording change but left as error since something did fail and it raises an error